trade-finance-manager-ui/server/controllers/utilisation-reports/helpers/reconciliation-for-report-helper.test.ts

```diff
@@ -127,31 +127,35 @@ describe('reconciliation-for-report-helper', () => {
         { formattedCurrencyAndAmount: 'EUR 300.00', dataSortValue: 1 },
       ];
 
-      it.each([
+      describe.each([
         { property: 'reportedFees' },
         { property: 'reportedPayments' },
         { property: 'totalReportedPayments' },
         { property: 'paymentsReceived' },
         { property: 'totalPaymentsReceived' },
-      ] as const)(
-        "correctly sorts and assigns the 'dataSortValue' and 'formattedCurrencyAndAmount' properties when converting the '$property' column",
-        ({ property }) => {
-          // Arrange
-          const feeRecordItems: FeeRecordItem[] = UNSORTED_CURRENCY_AND_AMOUNTS.map((currencyAndAmount) => ({
-            ...aFeeRecordItem(),
-            [property]: currencyAndAmount,
-          }));
+      ] as const)("when sorting and formatting the '$property' column", ({ property }) => {
+        // Arrange
+        const feeRecordItems: FeeRecordItem[] = UNSORTED_CURRENCY_AND_AMOUNTS.map((currencyAndAmount) => ({
+          ...aFeeRecordItem(),
+          [property]: currencyAndAmount,
+        }));
 
-          // Act
-          const feeRecordViewModelItems = mapFeeRecordItemsToFeeRecordViewModelItems(feeRecordItems);
+        // Act
+        const feeRecordViewModelItems = mapFeeRecordItemsToFeeRecordViewModelItems(feeRecordItems);
```

Comment:
I'm not a big fan of this refactor to be honest. My thoughts are as follows:
- `// Arrange` isn't needed above arrangement at the top of a describe block because everything there should just be arrangement anyway
- Anything crucial to the test, so in particular the `// Act` block should be in the test body so you never have to scroll for it, even if that means repeating it.
- The test name shouldn't just say 'this field should be this specific test value' but why, because otherwise if it fails I'll be like "cool this should be 5 but is 4 hmm err why should it be 5...". What I think we want is a test name along the lines "sets the dataSortValue for $property column to order the items alphabetically by currency and then numerically by amount in ascending order".
- The original body wasn't too bad. I appreciate the effort to write more specific assertions, but I think here we don't really get anything particularly more helpful, especially with the parameterisation making it a little unclear what we are reading.